engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-package-groups.html

```diff
@@ -0,0 +1,24 @@
+<span page-title ng-model="repository">{{ 'Package Groups for Repository:' | translate }} {{ repository.name }}</span>
+
+<a ui-sref="products.details.repositories.info({productId: product.id, repositoryId: repository.id})">
+  <i class="fa fa-double-angle-left"></i>
+  {{ "Back to Repository Details" | translate }}
+</a>
+
+<div data-extend-template="layouts/details-nutupane.html">
+  <div data-block="header">
+    <h3 translate>Package Groups in {{ repository.name }}</h3>
+  </div>
+  <div data-block="table">
+    <table class="table table-striped table-bordered" >
+      <thead>
+        <th bst-table-column><span translate>Name</span></th>
+      </thead>
+      <tbody>
+        <tr bst-table-row ng-repeat="packageGroup in detailsTable.rows">
+          <td bst-table-cell>{{ packageGroup.name }}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</div>
```

Comment:
I currently see it say '0 Selected' in the upper right corner.  I think you can fix this by adding an empty "selection-summary" data-block div.